reference/deprecation-notices.md

```diff
@@ -2,16 +2,25 @@
 
 This document contains a list of deprecation notices for Anbox Cloud and its components.
 
-## Ubuntu 20.04 (focal)
-
-Support for Ubuntu 20.04 (focal) is deprecated in 1.22.0 and is planned to be removed in Anbox Cloud 1.24.0.
+## Node controller
+*Deprecated in 1.23.0* ; *Removal in 1.24.0*
 
-## EmuGL
-
-Support for the EmuGL renderer is deprecated in 1.22.0 and planned to be removed in Anbox Cloud 1.23.0. Starting with 1.22.0, VirGL will be the default renderer for NVIDIA GPUs with driver version 545 and later.
+Use of node controller to manage the port forwarding from an instance to expose a service is deprecated in 1.23.0. Instead, the Anbox Management Service (AMS) will use a LXD proxy device to manage the port forwarding.
 
+(sec-appliance-deprecation)=
 ## Anbox Cloud Appliance
+*Deprecated in 1.22.0* ; *Removal in 1.24.2*
```

Comment:
1. I am not quite sure if the word `Deprecated in 1.22.0*` is correct here as we're NOT going to remove Anbox Cloud Appliance even after 1.24 release, people could still install appliance(v2) from the 1.24 or latest channel.
2. Why is the old appliance going to be removed from 1.24.2?

Reply:
1. Deprecated means it could be removed or it is not the latest supported version. So that's the right word.
2. Simon had mentioned this when we created this deprecation notice for 1.22.0. Could you confirm if this is still true - are we going to remove/stop support for the old appliance implementation from 1.24.2 @morphis?

Reply:
Without mentioning the epoch, when I read this, I thought the Anbox Cloud Appliance was going to be deprecated. However, that's not true; it's just the old appliance that is being deprecated. We will continue to provide features and maintenance for the new appliance moving forward. That's the original question I brought up here.

Reply:
Yes agreed, I have changed the heading to clarify